Makefile

```diff
@@ -14,5 +14,14 @@ test:
 clean:
 	rm -rf dist || true
 
+.PHONY: release
+# release by default will not publish. run with `publish=1` to publish
+goreleaserFlags = --skip-publish --snapshot
+ifdef publish
+	goreleaserFlags =
+endif
```

Comment:
for my own understanding: how does this work?

Reply:
I wanted to be extra careful with actually pushing things, so I created the `goreleaserFlags` make variable that defaults to `--skip-publish --snapshot` (which assures nothing scary is done), and the command that is run is appending this variable to the `goreleaser` command (line 24). but if the use defines the `publish` make variable (it can have any value, 1 is just an example) then the variable `goreleaserFlags` variable is emptied so nothing is appended to the `goreleaser` command, and the defaults are to publish.